lib/github/repository.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module GitHub
+  class Repository
+    class << self
+      def present?(client, full_name)
+        client.repository?(full_name)
+      end
```

Comment:
@tarebyte Sorry for this one. I thought we are talking about `GitHub::Repository.present?` so I put it into the `GitHub` module. Just got this fixed.